tests/phpunit/tests/kses.php

```diff
@@ -1110,6 +1110,31 @@ public function data_test_safecss_filter_attr() {
 				'css'      => 'height: expression( body.scrollTop + 50 + "px" )',
 				'expected' => '',
 			),
+			// RGBA background color are allowed.
+			array(
+				'css'      => 'background-color: rgba(0,0,0,0)',
+				'expected' => 'background-color: rgba(0,0,0,0)',
+			),
+			array(
+				'css'      => 'background-color: rgba(0, 0, 0, 0)',
+				'expected' => 'background-color: rgba(0, 0, 0, 0)',
+			),
+			array(
+				'css'      => 'background-color: rgba(100, 100, 100, 0)',
+				'expected' => 'background-color: rgba(100, 100, 100, 0)',
+			),
+			array(
+				'css'      => 'background-color: rgba(10%, 10%, 10%, 0)',
+				'expected' => 'background-color: rgba(10%, 10%, 10%, 0)',
+			),
+			array(
+				'css'      => 'background-color: rgba(0, 0, 0, 0.1)',
+				'expected' => 'background-color: rgba(0, 0, 0, 0.1)',
+			),
+			array(
+				'css'      => 'background-color: rgba(0, 0, 0, .1)',
+				'expected' => 'background-color: rgba(0, 0, 0, .1)',
+			),
```

Comment:
There's an test below under the comment "RGBA color values are not allowed." that will now fail.

It would be good to test for some malformed values too.

Reply:
I am thinking about regular expression and test cases, am I correct in this perception?

✅ `color: rgba(0, 0, 0 / 0.1)`
❌ `color: rgba(0, 0, 0/ 0.1)` (don't allow no space before slash)
❌ `color: rgba(0, 0, 0 /0.1)` (don't allow no space after slash)

✅ `color: rgba(0, 0, 0, 0)`
✅ `color: rgba(0,0,0,0)` (allow no space after comma)
❌ `color: rgba(0 , 0, 0, 0)` (don't allow space before comma)
❌ `color: rgba(0,   0, 0, 0)` (don't allow multiple spaces after comma *it would appear as single space on the comments)

✅ `color: rgba(100%, 0, 0, 0)`
❌ `color: rgba(100px,  0, 0, 0)` (don't allow units other than %)
❌ `color: rgba(red,  0, 0, 0)` (don't allow string value)

According to MDN, I believe the same rules can be applied to both [rgb()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) and [rgba()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgba).

Reply:
> I am thinking about regular expression and test cases, am I correct in this perception?

Yes, you are. These are the sort of tests I was suggesting. 

You may also wish to include something containing either a `"` or a `'` as an attempt to close the style tag. The expectation is that kses would return an empty string. 

Thanks so much.